### DISCUSSION TITLE
Is link metrics used with OT1.2 only ?

### DISCUSSION THREAD

**Discussion by testter2** [openthread/openthread#7753] [Q&A] [answered]
Hi,

I was just wondering for link metrics APIs as they are verifying that used version of openthread is equal 1.2 only.
My question is it intended that to be only 1.2 and not above it like 1.3 ?

this's the line of code I'm taking about:
`VerifyOrExit(neighbor->IsThreadVersion1p2(), error = kErrorNotCapable);
`
should it check for OT version to be equal or above OT1.2 ?

Thanks

**Comment by jwhui** [comment, marked as answer]
@testter2 , thanks for raising this question. The Link Metrics feature should apply to 1.2 and later versions, including 1.3. We should update the implementation accordingly.

**Reply by jwhui** [reply]
Just created issue #7754 and submitted PR #7755. Please review.

**Comment by testter2** [comment]
@jwhui Thanks for your quick response.
